src/rpc/rpc.cc

```diff
@@ -1198,6 +1199,10 @@ future<> server::connection::send_unknown_verb_reply(std::optional<rpc_clock_typ
       return isolation_config{};
   }
 
+  void server::warn_stuck_handler() {
+      seastar_logger.warn("Server cannot stop for too long, {} handlers are running", _handlers_running);
```

Comment:
I already see how annoying will it be to get such a message without it saying what handlers are running. We can have a counter per message type and print non zero values here.

Reply:
Good point